src/Components/Notifications/NotificationItem.tsx

```diff
@@ -31,16 +37,39 @@ const NotificationItem: React.FC<NotificationItemProps> = ({ item }) => {
   }
   const notificationTypeLabel = getNotificationType()
 
+  const markAsRead = async () => {
+    if (!relayEnvironment) {
+      return
+    }
+
+    try {
+      const response = await markNotificationAsRead(
+        relayEnvironment,
+        item.id,
+        item.internalID
+      )
+      const responseOrError = response.markNotificationAsRead?.responseOrError
+      const errorMessage = responseOrError?.mutationError?.message
+
+      if (errorMessage) {
+        throw new Error(errorMessage)
+      }
+    } catch (error) {
+      logger.error(error)
```

Comment:
**question / non-blocking**: What is our policy for handling exceptions / errors of frontend side? Should we send an error to Sentry here?

Reply:
when we call `logger.error', the error will be [logged in sentry](https://github.com/artsy/force/blob/80d11482e601df71506dac918ea8702d816eb0b7/src/Utils/logger.ts#L23-L25)